docs/modules/development/pages/rest/health_rest.adoc

```diff
@@ -0,0 +1,43 @@
+
+= Health Rest Service
+
+Service that allows health rest calls.
+
+Currently implemented on Minion and Sentinel by default.
+
+Examples:
+
+* /rest/health
+* /rest/health/probe
+* /rest/health?tag=local
+* /rest/health/probe?tag=local
+
+== GETs (Reading Data)
+
+[options="header", cols="5,10"]
+|===
+| Resource             | Description
+| `/health`            | Health check results for components. Include health check result details for each element.
+| `/health/prob`       | Health check probe for components. Only indicate if the health is good or not. 
+|===
+
+The following table shows all supported query string parameters and their default values.
+
+[options="header", cols="1,3,1"]
+|===
+| Name               | Description                  | Default 
+| t                 | Timeout in milliseconds      | 5000 
+| maxAgeMs          | Max age to live miliseconds  | 90000 
+| tag                     
+| Use tag to filter Health Check components. A tag could be: +
+ * local + 
+ * rest +
+ * telemetry +
+ * dao +
+ * minion +
+ * elastic +
+ * bundle 
+                                                    | '' 
+|===
+
```

Comment:
Also add documentation for mixing multiple tags.  

Also define error http codes and when user may get those error codes.

Reply:
Not sure about the errors. from what I can see there will be only 200 or 404. Are you referring to the Status enum ?